exporter/awsemfexporter/datapoint.go

```diff
@@ -213,42 +235,122 @@ func (dps histogramDataPointSlice) IsStaleNaNInf(i int) (bool, pcommon.Map) {
 }
 
 // CalculateDeltaDatapoints retrieves the ExponentialHistogramDataPoint at the given index.
+// As CloudWatch EMF logs allows in maximum of 100 target members, the exponential histogram metric are split into two data points if the total buckets exceed 100,
+// the first data point contains the first 100 buckets, while the second data point includes the remaining buckets.
+// Re-calculated Min, Max, Sum, Count for each split:
+// 1. First split datapoint:
+//   - Max: From original metric.
+//   - Min: Last bucket’s bucketBegin in the first split.
+//   - Sum: From original metric.
+//   - Count: Calculated from the first split buckets.
+//
+// 2. Second split datapoint:
+//   - Max: First bucket’s bucketEnd in the second split.
+//   - Min: From original metric.
+//   - Sum: 0.
```

Comment:
Why is the `Sum` set to 0 in this case? Shouldn't the two datapoints be essentially separate?

Reply:
The reason why we are not calculating Sum base on bucket datapoint: 
the raw Sum is using the raw metric data. Since the otel collector doesn’t have visibility of metric raw data,  if we recalculate the sum values for the split datapoint, we can only use the approximate value (mid value for each bucket) which might have an offset of the real value


In this case:
I'm only assigning actual `Sum` to the first split, all the following split will have sum equals to 0. Then, after emf log aggregation, the total sum of the datapoints is exactly same before split.